### DISCUSSION TITLE
command line run revChatGPT notion happens

### DISCUSSION THREAD

**Discussion by geekaiL** [acheong08/ChatGPT#127] [Show and tell]
After typing in the command line, nothing happens
command line shows that it has been executed
please

**Comment by acheong08** [comment]
Press enter twice to ask a question

**Reply by acheong08** [reply]
Or is it Python issue? Does no output show at all?

**Reply by geekaiL** [reply]
Yes，no output show at all，please help，thanks

**Reply by acheong08** [reply]
Open an issue, not discussion. Follow bug reporting template

**Reply by geekaiL** [reply]
sorry
